docs/content/latest/admin/yb-tserver.md

```diff
@@ -25,53 +25,283 @@ $ ./bin/yb-tserver \
-`--webserver_doc_root` | N | The `www` directory in the YugabyteDB home directory | Monitoring web server home.
-`--cql_proxy_bind_address` | N | `0.0.0.0:9042` | YCQL API bind address.
-`--cql_proxy_webserver_port` | N | 12000 | YCQL metrics monitoring port
-`--redis_proxy_bind_address` | N | `0.0.0.0:6379` | YEDIS API bind address.
-`--redis_proxy_webserver_port` | N | 11000 | YEDIS metrics monitoring port.
-`--start_pgsql_proxy` | N | N/A | Enable YSQL API.
-`--pgsql_proxy_bind_address` | N | `0.0.0.0:5433` | YSQL API bind address.
-`--pgsql_proxy_webserver_port` | N | 13000 | YSQL metrics monitoring port.
-`--placement_zone` | N |`rack1` | Name of the availability zone or rack where this instance is deployed.
-`--placement_region` | N |`datacenter1` | Name of the region or data center where this instance is deployed.
-`--placement_cloud` | N |`cloud1` | Name of the cloud where this instance is deployed.
-`--logtostderr` | N | N/A  | Log to standard error.
-`--flagfile` | N | N/A  | Load flags from the specified file.
-`--version` | N | N/A | Show version and build info.
-`--use_cassandra_authentication` | N | false | If enabled, it will require YCQL client authentication (username/password), enable YCQL security statements (CREATE/DROP/GRANT/REVOKE ROLE and GRANT/REVOKE PERMISSION), and enforce permissions for YCQL statements.
-`--rocksdb_compact_flush_rate_limit_bytes_per_sec` | N | 256MB | Used to control rate of memstore flush and SSTable file compaction.
-`--remote_bootstrap_rate_limit_bytes_per_sec` | N | 256MB | Rate control across all tablets being remote bootstrapped from or to this process.
-`--yb_num_shards_per_tserver` | N | -1 | The number of shards per yb-tserver per table when a user table is created. Server automatically picks a valid default internally.
-`--durable_wal_write` | N | false | If set to `false`, the writes to the Raft log are synced to disk every `interval_durable_wal_write_ms` milliseconds or every `bytes_durable_wal_write_mb` MB, whichever comes first. This default setting is recommended only for multi-AZ or multi-region deployments where the zones/regions are independent failure domains and there isn't a risk of correlated power loss. For single AZ deployments, this flag should be set to `true`.
-`--interval_durable_wal_write_ms` | N | 1000ms | When `durable_wal_write` is false, writes to the Raft log are synced to disk every `interval_durable_wal_write_ms` milliseconds or `bytes_durable_wal_write_mb` MB whichever comes first.
-`--bytes_durable_wal_write_mb` | N | 1MB | When `durable_wal_write` is false, writes to the Raft log are synced to disk every `bytes_durable_wal_write_mb` MB or `interval_durable_wal_write_ms` milliseconds whichever comes first.
+## Syntax
+
+```sh
+yb-tserver [ options ]
+```
+
+## Configuration options
+
+### General options
+
+#### --help
+
+Displays help on all options.
+
+#### --helpon
+
+Displays help on modules named by the specified option (or flag) value.
+
+#### --flagfile
```

Comment:
We should move this below the mandatory options.

Reply:
Good catch.